### PR TITLE
Room app

### PR DESCRIPTION
**Pull request by jlswift2**
Please be as critical as possible!

### CODE REVIEW

File: rooms/models.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+
+
+class Room(models.Model):
+    name = models.CharField(max_length=50)
+    contents = models.TextField()
+    maxCapacity = models.PositiveIntegerField(validators=[MinValueValidator(1), MaxValueValidator(350)])
```
Comment (piebug) [inline rooms/models.py]: Can we change this to `capacity`? I'm a picky bitch, and technically `maxCapacity` is redundant

**Review by piebug — COMMENTED** [review]
Oh, and one more comment! You should commit the migration files that are generated when you run `python manage.py makemigrations`. Everyone who runs the project will use those

File: rooms/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (piebug) [inline rooms/admin.py]: OH, and let's go ahead and register the new model in the admin app

**Comment by jlswift2** [comment]
I believe I have fixed all that was requested. Mind taking another look?

**Review by piebug — APPROVED** [review]
Looks good, my dude!

**Pull request merged**